modules/runner/src/integrationTest/sql/types/blob/test_blob.test

```diff
@@ -62,46 +62,12 @@ statement ok
 DELETE FROM blobs
 
 # Implicit cast
-statement ok
+statement error
 INSERT INTO blobs VALUES('blablabla')
 
-# BINARY with “non-printable” octets
-statement ok
-INSERT INTO blobs VALUES('abc �'::VARBINARY)
-
-query T
-SELECT b::varchar FROM blobs ORDER BY b
-----
-abc �
-blablabla
-
 # BINARY null and empty values
+statement error
 query T
 SELECT ''::VARBINARY
-----
-(empty)
 
-query T
-SELECT NULL::VARBINARY
-----
-NULL
```

Comment:
This test looks valid.

Reply:
Seems it`s possible to cast NULL to all types, need to append additional test ....